package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "eleventy-netlify-boilerplate",
+  "version": "1.0.2",
+  "description": "A boilerplate for building a fast static website with the Eleventy static site generator, for deployment to Netlify.",
+  "scripts": {
+    "build": "npx eleventy",
+    "watch": "npx eleventy --watch",
+    "serve": "npx eleventy --serve",
+    "debug": "DEBUG=* npx eleventy",
+    "sass": "node-sass --watch _includes/assets/sass/styles.scss _includes/assets/css/styles.css"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/danurbanowicz/eleventy-netlify-boilerplate"
```

Comment:
Let's update the author and the repo urls here

Reply:
@lewisnyman Updated